plugins/plugin-site/src/components/PluginActiveWarnings.jsx

```diff
@@ -21,7 +21,7 @@ function PluginActiveWarnings({securityWarnings}) {
 function multipleWarnings(active) {
     return (
         <div>
-            {'The current version of this plugin contains multiple vulnerabilities:'}
+            {'The Jenkins project announced security vulnerabilities affecting the current version of this plugin:'}
```

Comment:
It might help to add "unresolved" here to make it clearer that there is no fix and to align it with  https://www.jenkins.io/security/plugins/#unresolved for clarity.

```suggestion
            {'The Jenkins project announced unresolved security vulnerabilities affecting the current version of this plugin:'}
```